### PR TITLE
feat: user profile screen

### PR DESCRIPTION
**Pull request by TikuGaelle**
## Description
 User profile screen.
  Solves issue #83 

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read and ran all relevant commands as specififed in the Running Tests section of the [Contributor Guide].
- [ ] The title of the PR follows the [Conventional Commits] guideline
- [ ] My local branch follows the naming standards in the [Deepsource Branch Naming Convention] or [Biodiversity Branch Naming Convention]
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy],
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

[Contributor Guide]: https://github.com/FlutterPlaza/.github/blob/main/CONTRIBUTING.md
[Conventional Commits]: https://www.conventionalcommits.org/en/v1.0.0-beta.4/
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[Biodiversity Branch Naming Convention]: https://bit.ly/3DyYSwM
[Deepsource Branch Naming Convention]: https://bit.ly/3Y08Gs4

### CODE REVIEW

File: lib/core/presentation/theming/colors/colors.dart [hunk 1]
```diff
@@ -8,7 +8,7 @@ class _AppColors {
   static Color onErrorW = const Color(0xffFF000D);
   // static Color bgColorW = const Color(0xffF7F7F7);
   static Color bgColorW = Color(0xffFFFFFF);
-  static Color onBgColorW = Color(0xffD9D9D9);
+  static Color onBgColorW = Color(0xffF5F5F5);
```
Comment (fonkamloic) [inline lib/core/presentation/theming/colors/colors.dart:11]: Have you tested the hold app to see how this color change affects the app?

Comment (TikuGaelle) [reply lib/core/presentation/theming/colors/colors.dart:11]: No. I changed that to make the grey texts visible on the background color.

Comment (fonkamloic) [reply lib/core/presentation/theming/colors/colors.dart:11]: Does it match the colors on Figma?

Comment (TikuGaelle) [reply lib/core/presentation/theming/colors/colors.dart:11]: Yes.

File: lib/profile/view/profile_page.dart [hunk 2]
```diff
@@ -16,163 +17,53 @@ class _ProfileScreenState extends State<ProfileScreen> {
...
           return Padding(
             padding: EdgeInsets.only(
-              top: box.maxHeight * 0.1,
-              left: box.maxWidth * 0.03,
-              right: box.maxWidth * 0.03,
+              top: box.maxHeight * 0.09,
+              left: box.maxHeight * 0.03,
+              right: box.maxHeight * 0.03,
             ),
             child: Column(
```
Comment (lkmandy) [inline lib/profile/view/profile_page.dart]: - Wrap your column in a `SingleChildScrollView()` to resolve the bottom overflow
- The Profile title should be increased too, `size: 32px, weight: 600 color: #181818`
- Also the screen background color should be htis shade of white `#F7F7F7` as on the design

pr-rv2

**Review by lkmandy — CHANGES_REQUESTED** [review]

**Review by fonkamloic — APPROVED** [review]

**Pull request merged**